src/test/java/teammates/test/cases/ui/StudentProfileEditSaveActionTest.java

```diff
@@ -8,12 +8,17 @@
 
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.StudentAttributes;
 import teammates.common.datatransfer.StudentProfileAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
 import teammates.common.util.Const;
 import teammates.common.util.FieldValidator;
 import teammates.common.util.StringHelper;
 import teammates.test.driver.AssertHelper;
 import teammates.ui.controller.RedirectResult;
+import teammates.ui.controller.ShowPageResult;
+import teammates.ui.controller.StudentCourseDetailsPageAction;

```

Comment:
We are testing that if a user uploaded his profile pic, that should be visible on his/her team member's `teamView` page. The reason is as stated [above](https://github.com/TEAMMATES/teammates/pull/5948#issuecomment-237472747) by chowyb.